container/src/main/java/com/flipkart/gap/usl/container/resource/RetrievalResource.java

```diff
@@ -68,13 +70,14 @@ public Response getDimensionForEntity(@NotNull @PathParam("entityId") String ent
     @ApiOperation("To fetch the response of the Dimension list")
     @ApiResponses(value = {@ApiResponse(code = HttpStatus.SC_NO_CONTENT, message = "Dimension not exists for the given entity")})
     public Response getDimensionList(@NotNull @PathParam("entityId") String entityId,
-                                     @NotNull List<String> dimensionsToFetch) {
```

Comment:
Rename Request class to something specific like FetchBulkDimension

Reply:
I was using the same Request class which we used for bulk entity id API, I was thinking that there are very few APIs in USL and changes will be very few so we can use the same generic class in all APIs.

I can still do the changes if you think that will be better